Frontend/src/locales/en.json

```diff
@@ -52,17 +52,17 @@
   },
   "settings": {
     "changeMail": "Change your E-mail here.",
-    "changeUsername": "Change your username here",
-    "changePassword":"Change your password here",
-    "changeProfilePicture": "change your profile pic here",
+    "changeUsername": "Change your username here.",
+    "changePassword":"Change your password here.",
+    "changeProfilePicture": "Change your profile picture here",
     "deleteQuestion": "Delete your Account?",
-    "deleteAccount": "delete",
+    "deleteAccount": "DELETE",
     "oldPassword": "old password",
     "newPassword": "new password",
-    "deleteSure": "Are you sure?",
+    "deleteSure": "Are you sure you want to delete your account?",
     "deleteHint" : "Deleting your account is permanent and irreversible",
```

Comment:
Deleting your account is permanent and irreversible! All your data and pictures will be deleted permanently.